src/mscorlib/src/System/Globalization/CultureData.Unix.cs

```diff
@@ -14,6 +14,20 @@ namespace System.Globalization
 {
     internal partial class CultureData
     {
+        private static bool s_invariantGlobalizationMode = InitGlobalizationInvariantMode();
```

Comment:
Right, that's why it would be best to cache the bool in instance fields where possible - instance fields are the fastest to access.